api/app/src/Entity/StagingSelectedCandidate.php

```diff
@@ -40,7 +40,7 @@ class StagingSelectedCandidate
      *
      * @ORM\SequenceGenerator(sequenceName="candidate_id_seq", allocationSize=1, initialValue=1)
      */
-    public int $id;
+    public ?int $id = null;
```

Comment:
should this ever be null, as it's autogenerated?

Reply:
I had to make the property nullable as it was failing on an integration test because the csv/serializer was expecting a value when converting csv data into an object. I set it as nullable so that the serializer doesn't expect id to be set and id can still be auto-generated by Doctrine once the object is persisted.